### PR TITLE
OSDOCS-1868: Added OVN and cluster maximum release notes

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1868
https://issues.redhat.com/browse/PERFSCALE-855

Preview build: https://deploy-preview-33221--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-scale

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@chaitanyaenr @mffiedler PTAL

**Review by jtaleric — APPROVED** [review]
LGTM - Thanks!

**Review by kedark3 — APPROVED** [review]
LGTM :tada: thx

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -275,6 +275,18 @@ The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled
...
 === Scale
 
+[id="ocp-4-8-scale-cluster-maximums"]
+==== Cluster maximums
+
+Updated guidance around xref:../scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc#planning-your-environment-according-to-object-maximums[cluster maximums] for {product-title} {product-version} is now available.
+
+[IMPORTANT]
+====
+No large scale testing for performance against Open Virtual Network (OVN) Kubernetes testing was executed for this release.
```
Comment (jboxman) [inline release_notes/ocp-4-8-release-notes.adoc:285]: @ahardin-rh, fyi, we're calling this OVN-Kubernetes throughout the networking docs, if this is about the network provider itself: https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html

**Review by chaitanyaenr — APPROVED** [review]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-8-release-notes.adoc:285]: @jboxman Thank you for pointing this out! I will update the wording to align.

**Pull request merged**